src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -0,0 +1,192 @@
+package frc.robot.subsystems.conveyor;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkLimitSwitch;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.RobotMap;
+import frc.robot.utils.SparkMaxUtils;
+
+public class Conveyor extends SubsystemBase {
+  private CANSparkMax frontMotor =
+      new CANSparkMax(RobotMap.FRONT_CONVEYOR_CAN_ID, MotorType.kBrushless);
+  private CANSparkMax backMotor =
+      new CANSparkMax(RobotMap.BACK_CONVEYOR_CAN_ID, MotorType.kBrushless);
+
+  /** Configured to read inches, positive towards the shooter. */
+  private RelativeEncoder frontMotorEncoder = frontMotor.getEncoder();
+  /** Configured to read inches, positive towards the shooter. */
+  private RelativeEncoder backMotorEncoder = backMotor.getEncoder();
+
+  /**
+   * These aren't actually limit switches; we just use SparkLimitSwitch objects to access them
+   * easily. At the moment, these are unused. TODO: If we use these, add them to Shuffleboard.
+   */
+  SparkLimitSwitch
+      beamBreakSensorOne = frontMotor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen),
+      beamBreakSensorTwo = frontMotor.getReverseLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen),
+      beamBreakSensorThree = backMotor.getForwardLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen),
+      beamBreakSensorFour = backMotor.getReverseLimitSwitch(SparkLimitSwitch.Type.kNormallyOpen);
+
+  /**
+   * Used to define the current position of the note.
+   *
+   * <ul>
+   *   <li>NO_NOTE: There is no note in the conveyor.
+   *   <li>HOLDING_NOTE: The conveyor is holding a note in place.
+   *   <li>PARTIAL_NOTE: The conveyor is in the process of receiving a note from the intake, sending
+   *       a note to the shooter, or scoring a note in the trap or amp.
+   * </ul>
+   */
+  public enum ConveyorPositions {
```

Comment:
this should be singular, ConveyorPosition

Reply:
fixed!